config/legacy/parameters.yaml.dist

```diff
@@ -59,6 +59,14 @@ parameters:
         tiqr:
             loa: 3
 
+    second_factors_display_name:
+      yubikey: Yubikey
+      azuremfa: AzureMFA
+      webauthn: WebAuthn
+      tiqr: Tiqr
+      demo_gssp: GSSP Demo
+      demo_gssp_2: GSSP Demo 2
+
     # Sets the number of tokens allowed for each identity.
```

Comment:
Do we want these to be traslatable? E.g.
```
second_factors_display_name:
    yubikey: 
        nl_NL: Yubikey
        en_GB: Yubikey
    azuremfa: 
        nl_NL: Azure MFA
        en_GB: Azure MFA
```

Reply:
In the pivotal story @phavekes mentioned: `Token display names will not be language specific`.

Reply:
Yes, these wil probably never be language-specific, so let's not introduce extra complexity